tests/consts.sh

```diff
@@ -6,7 +6,7 @@
 
 # Released version from GitHub Releases.
 # e.g. '21.1.2'
-OASIS_CORE_VERSION='21.2.3'
+OASIS_CORE_VERSION='21.2.4'
```

Comment:
You should also update the `BUILD_NUMBER` below to correspond to the Buildkite build of 21.2.4.